public/javascripts/searchsession.js

```diff
@@ -55,10 +55,22 @@ function slide(keyword, pageNumber) {
                         }
 
                         if (sessions[session].completed) {
-                            usersFound += "<td><div><span class='label label-default' >Completed</span></div><td></tr>";
+                            usersFound += "<td><div><span class='label label-default' >Completed</span></div></td>";
                         } else {
-                            usersFound += "<td><div><span class='label label-warning' >Pending</span><br/></div><td></tr>";
+                            usersFound += "<td><div><span class='label label-warning' >Pending</span><br/></div></td>";
                         }
+                        if(sessions[session].completed && !sessions[session].cancelled) {
```

Comment:
It looks like there's a space required in between 
`if( `
and
`}else`

I know this is too much attention to details but reformatting using IDE would automatically fix these issues.